shell/packages/sandstorm-accounts-ui/login_buttons.js

```diff
@@ -2,6 +2,9 @@
 var loginButtonsSession = Accounts._loginButtonsSession;
 
 var helpers = {
+  isCurrentRoute: function (routeName) {
+    return Router.current().route.getName() == routeName;
```

Comment:
I think I wrote the original code, so I'm probably the guilty party here, sorry.

I am curious what we should be doing instead - inject an instance of the Router into everything that might care about it?  (I've never actually seen an instance of iron router, only the global Router.)